### PR TITLE
chore: prettier eslint versionup

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

* add prettier
* eslint: 4.18.2 -> 4.19.1

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Comment by jung-han** [comment]
Can one of the admins verify this patch?

File: .eslintrc.js [hunk 1]
```diff
@@ -1,24 +1,27 @@
 module.exports = {
-  'extends': 'tui/es6',
-  'parserOptions': {
-      "sourceType": "module"
+  extends: ['tui/es6', 'plugin:prettier/recommended'],
+  plugins: ['prettier'],
```
Comment (jung-han) [inline .eslintrc.js]: * prettier 플러그인 추가

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (jung-han) [inline .prettierrc]: 부서 prettier 룰 적용

File: package.json [hunk 3]
```diff
@@ -74,7 +75,10 @@
     "typescript": "^3.2.2",
     "webpack": "^1.15.0",
     "webpack-dev-server": "^1.16.5",
-    "webpack-polyfills-plugin": "^0.2.0"
+    "webpack-polyfills-plugin": "^0.2.0",
+    "prettier": "^1.19.1",
+    "eslint-config-prettier": "^6.9.0",
+    "eslint-plugin-prettier": "^2.7.0"
```
Comment (jung-han) [inline package.json]: 의존 추가

File: tslint.json [hunk 4]
```diff
@@ -7,7 +8,12 @@
     ],
     "trailing-comma": false,
     "max-classes-per-file": false,
-    "no-namespace": false
+    "no-namespace": false,
+    "no-empty": false,
+    "arrow-parens": false,
+    "no-console": false,
+    "object-literal-sort-keys": false,
+    "no-unused-expression": false
```
Comment (jung-han) [inline tslint.json]: `test/types` 깨지는 부분 수정. 이후에 eslint버전이 5 이상 올라간다면 tslint를 제거하고 eslint로 통합해야 합니다.

**Comment by junghwan-park** [comment]
Test this please.

**Comment by junghwan-park** [comment]
Test this please

**Comment by jinwoo-kim-nhn** [comment]
수고하셨습니다!! ㅎㅎ

**Comment by jung-han** [comment]
ok to test

**Pull request merged**